index.html

```diff
@@ -1,48 +1,285 @@
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="about-div">
+        <h1 class="about-heading">About me</h1>
+        <p class="about-descrp">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="about-btn">Get My Resume</button>
+      </div>
+      <div class="languages">
+        <h3 class="lang-heading">
+          <i class="fa-solid fa-diamond"></i>Languages
+        </h3>
+        <ul class="lang-ul">
+          <li class="lang-li">
+            <h5 class="lang-p">javascript</h5>

```

Comment:
[Optional]
- [ ] It's better if technologies/languages/tools names are capitalized 👍